docs/for_developers.md

```diff
@@ -43,15 +43,30 @@ flowchart LR
 
     To simply init download process we have to run following commands:
 
-    ```bash
-    poetry run ./yaptide/admin/simulators.py download-shieldhit --dir bin
-    ```
+    === "Linux"
+
+        ```bash
+        poetry run ./yaptide/admin/simulators.py download-shieldhit --dir bin
+        ```
+
+    === "Windows (PowerShell)"
+        ```powershell
+        poetry run .\yaptide\admin\simulators.py download-shieldhit --dir bin
+        ```
```

Comment:
```suggestion
    === "Linux"

        ```bash
        poetry run yaptide/admin/simulators.py download-shieldhit --dir bin
        ```

    === "Windows (PowerShell)"
        ```powershell
        poetry run yaptide\admin\simulators.py download-shieldhit --dir bin
        ```
```

Reply:
Does it work without `./` or `.\` ?

Reply:
See also https://github.com/yaptide/yaptide/pull/917#issuecomment-2059833535